website/docs/r/hdinsight_spark_cluster.html.markdown

```diff
@@ -251,7 +251,23 @@ A `zookeeper_node` block supports the following:
 
 * `virtual_network_id` - (Optional) The ID of the Virtual Network where the Zookeeper Nodes should be provisioned within. Changing this forces a new resource to be created.
 
---- 
+---
+
+A `display_encryption_properties` block supports the following:
+
+* `encryption_algorithm` - (Optional) This is an algorithm identifier for encryption. Possible values are `RSA1_5`, `RSA-OAEP`, `RSA-OAEP-256`.
+
+* `encryption_at_host` - (Optional) This is indicator to show whether resource disk encryption is enabled.
+
+* `key_name` - (Optional) This is the name of key used for enabling disk encryption.
+
+* `key_version` - (Optional) This is the specific version of the key used for enabling disk encryption.
```

Comment:
these are key vault items?
```suggestion
* `key_vault_key_name` - (Optional) This is the name of key used for enabling disk encryption.

* `key_vault_key_version` - (Optional) This is the specific version of the key used for enabling disk encryption.
```

also could you not combine these into `key_vault_key_id` which takes the versionless or versioned id?